Description: Fix 32-bit builds.
 This fixes a compilation error in lifting-gb.c where we tried to cast
 a pointer to a long to a pointer to an int64_t, which doesn't work on 32-bit
 systems.
Author: Doug Torrance <dtorrance@debian.org>
Forwarded: https://github.com/algebraic-solving/msolve/pull/172
Last-Update: 2024-07-30

--- a/src/msolve/msolve-data.h
+++ b/src/msolve/msolve-data.h
@@ -352,7 +352,7 @@
 
 typedef struct{
   int dim;
-  long dquot;
+  int64_t dquot;
   int32_t ht_size; /* initial_hts */
   int32_t nr_threads;
   int32_t max_nr_pairs;
